datareservoirio/_logging.py

```diff
@@ -28,11 +28,20 @@ def wrapper(self, *args, **kwargs):
         try:
             return func(self, *args, **kwargs)
         except Exception as e:
-            properties = {
-                "custom_dimensions": {
-                    "drio_package": f"python-datareservoirio/{drio.__version__}"
+            if os.getenv(ENV_VAR_ENGINE_ROOM_APP_ID) is not None:
+                engine_room_app_id = os.environ[ENV_VAR_ENGINE_ROOM_APP_ID]
+                properties = {
+                    "custom_dimensions": {
+                        "drio_package": f"python-datareservoirio/{drio.__version__}",
+                        "EngineRoom App ID": engine_room_app_id,
```

Comment:
I also have doubts when it comes to syntax of that if-else. Instead of doing copy-paste of properties and custom dimensions, maybe it's possible to create properties with drio-package dimension first and then, use  `if os.getenv(ENV_VAR_ENGINE_ROOM_APP_ID) is not None`, to decide whether to add key-value entry with ER app id to properties.custom_dimensions? Is there any `.add()` method on `properties.custom_dimensions` object? It would be way prettier if it looks like this:

```
properties = {
    "custom_dimensions": {
        "drio_package": f"python-datareservoirio/{drio.__version__}"
      }
if os.getenv(ENV_VAR_ENGINE_ROOM_APP_ID) is not None:
    properties.custom_dimensions.add("engineRoomAppId", os.getenv(ENV_VAR_ENGINE_ROOM_APP_ID))
```

But I don't know if that's possible in Python..